src/semgrep_agent/main.py

```diff
@@ -122,6 +122,15 @@ def main(
     formatter.dump(results.findings.new)
     obj.sapp.report_results(results)
 
-    exit_code = 1 if results.findings.new else 0
+    exit_code = get_error_code(results)
     click.echo(f"=== exiting with {'failing' if exit_code == 1 else 'success'} status")
```

Comment:
Could you also print the number of results that were not printed? Something like the sample output in https://github.com/returntocorp/semgrep-action/issues/34#issue-676927729